specs/protocol/predeploys.md

```diff
@@ -39,7 +39,7 @@ network forking.
 
 Predeploy addresses exist in 1 byte namespace `0x42000000000000000000000000000000000000xx`.
 Proxies are set at each possible predeploy address except for the
-`GovernanceToken` and the `ProxyAdmin`.
+`GovernanceToken`.
```

Comment:
I believe this should be updated, i believe weth has no proxy

Reply:
Yes, double checked here 
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Predeploys.sol#L141-L145

and updated here fce5a999ca87c33d8606a3721c5ae6a96827f8cd.